src/frontend/js/pages/DashboardOrderApplication/index.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, screen } from '@testing-library/react';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { IntlProvider } from 'react-intl';
+import fetchMock from 'fetch-mock';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { RichieContextFactory as mockRichieContextFactory } from 'utils/test/factories/richie';
+import { Order } from 'types/Joanie';
+import { OrderFactory, TargetCourseFactory } from 'utils/test/factories/joanie';
+import { mockProduct } from 'utils/test/mockProduct';
+import { LearnerDashboardPaths } from 'widgets/Dashboard/utils/learnerRouteMessages';
+import { createTestQueryClient } from 'utils/test/createTestQueryClient';
+import { SessionProvider } from 'contexts/SessionContext';
+import { DashboardTest } from 'widgets/Dashboard/components/DashboardTest';
+import { expectBreadcrumbsToEqualParts } from 'utils/test/expectBreadcrumbsToEqualParts';
+import { expectUrlMatchLocationDisplayed } from 'utils/test/expectUrlMatchLocationDisplayed';
+
+jest.mock('utils/context', () => ({
+  __esModule: true,
+  default: mockRichieContextFactory({
+    authentication: { backend: 'fonzie', endpoint: 'https://demo.endpoint' },
+    joanie_backend: { endpoint: 'https://joanie.endpoint' },
+  }).generate(),
+}));
+
+describe('<DashboardOrderApplication />', () => {
+  const WrapperWithDashboard = (route: string) => {
+    return (
+      <QueryClientProvider client={createTestQueryClient({ user: true })}>
+        <IntlProvider locale="en">
+          <SessionProvider>
+            <DashboardTest initialRoute={route} />
```

Comment:
I think we should directly use the page component  `DashboardOrderApplication` here instead of loading the whole router.

Reply:
The goal is to use the router to see how the sidebar behaves here